subprojects/internal-performance-testing/internal-performance-testing.gradle.kts

```diff
@@ -95,3 +95,7 @@ tasks.jar {
     from(files(deferred{ flamegraph.map { zipTree(it) } }))
 }
 
+configurations.all {
+    resolutionStrategy.cacheDynamicVersionsFor(1, TimeUnit.MINUTES)
+}
```

Comment:
How does this help?

Reply:
I was hoping it helps while I am using snapshot version that CI picks up the newest published `1.0-SNAPSHOT` version, instead of whatever there is the local dependency cache. This needs to be remove once we published a non-changing Gradle profiler version (1.0?).

Reply:
Removed.